### PR TITLE
adding a constraint on the image height in the modal media

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

File: MediaAdminBundle/Resources/public/less/mediaModal.less [hunk 1]
```diff
@@ -38,6 +38,12 @@
                     width: 741px;
                     height: calc(~"100% - 55px");
 
+                    form[name=media_crop] {
+                        max-height: 100%;
+                        overflow-y: scroll;
```
Comment (ngilain) [inline MediaAdminBundle/Resources/public/less/mediaModal.less]: I think the ticket requires a redim, not a scroll, isn't it ?

File: MediaAdminBundle/Resources/public/less/mediaModal.less [hunk 2]
```diff
@@ -38,6 +38,15 @@
                     width: 741px;
                     height: calc(~"100% - 55px");
 
+                    form[name=media_crop] {
+                        position: relative;
+                        height: inherit;
+
+                        #preview_thumbnail {
+                            max-height: 80%;
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/less/mediaModal.less]: why 80% and not 100% ?

Comment (SOLenG) [inline MediaAdminBundle/Resources/public/less/mediaModal.less]: I went over 80% for the other inputs in the same display (without scroll bar)

File: MediaAdminBundle/Resources/public/less/mediaModal.less [hunk 3]
```diff
@@ -38,6 +38,16 @@
                     width: 741px;
                     height: calc(~"100% - 55px");
 
+                    form[name=media_crop] {
+                        position: relative;
+                        height: inherit;
+
+                        #preview_thumbnail {
+                            min-height: 100px;
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/less/mediaModal.less]: why add a min-height

Comment (SOLenG) [inline MediaAdminBundle/Resources/public/less/mediaModal.less]: Otherwise the image can be resized up to disappearance.

**Comment by SOLenG** [comment]
@itkg-alavieille pr fixed

**Comment by alavieille** [comment]
:+1:

**Pull request merged**